backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -235,31 +231,70 @@ def generate_destruction_report(self) -> None:
             "zaaktype omschrijving",
             "selectielijst procestype nummer",
         ]
+
+        worksheet = workbook.add_worksheet(name=gettext("Deleted zaken"))
+        worksheet.write_row(0, 0, column_names)
+
+        for row_count, item in enumerate(
+            self.items.filter(processing_status=InternalStatus.succeeded).iterator(
+                chunk_size=1000
+            )
+        ):
+            data = [
+                item.extra_zaak_data["url"],
+                item.extra_zaak_data["einddatum"],
+                item.extra_zaak_data["resultaat"],
+                item.extra_zaak_data["startdatum"],
+                item.extra_zaak_data["omschrijving"],
+                item.extra_zaak_data["identificatie"],
+                item.extra_zaak_data["zaaktype"]["url"],
+                item.extra_zaak_data["zaaktype"]["omschrijving"],
+                item.extra_zaak_data["zaaktype"]["selectielijst_procestype"]["nummer"],
+            ]
+
+            worksheet.write_row(row_count + 1, 0, data)
+
+    def _make_review_process_sheet(self, workbook: xlsxwriter.Workbook) -> None:
+        column_names = [
+            gettext("Group"),
+            gettext("Name"),
+            gettext("Date/Time"),
+            gettext("Changes"),
+        ]
+
+        worksheet = workbook.add_worksheet(name=gettext("Review process"))
+        worksheet.write_row(0, 0, column_names)
+
+        logs = TimelineLog.objects.for_object(self).filter(
+            template="logging/destruction_list_reviewed.txt", extra_data__approved=True
+        )
+        for row_count, log in enumerate(logs):
+            user_data = log.extra_data["user"]
+            # Not using the FK because the user might have been deleted in the mean time
+            formatted_user = f"{user_data["first_name"]} {user_data["last_name"]} ({user_data["username"]})"
```

Comment:
Maybe introduce (template) filters which we can use in the templates and in in python code so we don't keep multiple implementations of this?

Reply:
I realised that the templates are still using the FK user instead of the data in `extra_data`, so I will open an issue to refactor this :S 

I have made a reusable method for now for formatting the user that we can use then.

Reply:
https://github.com/maykinmedia/open-archiefbeheer/issues/530